codec/reflectcodec/type_codec.go

```diff
@@ -89,8 +89,12 @@ func (c *genericCodec) Size(value interface{}) (int, error) {
 	return size, err
 }
 
-// size returns the size of the value along with whether the value is constant sized.
 func (c *genericCodec) size(value reflect.Value) (int, bool, error) {
+	return c.sizeWithOmitEmpty(value, false)
```

Comment:
That's fine. Because `omitEmpty: true` is passed by [struct properties ](https://github.com/ava-labs/avalanchego/pull/2171/files/1ad3e42ef00d4fca4c7c80ef1dca7d332771c8b0#diff-d0a849d09d3e81f7c9a022d1371b55169cac9fa301324bee1ed2d06da010e528R219)